examples/language-sdk-instrumentation/rust/README.md

```diff
@@ -1,2 +1,148 @@
 ## Continuous Profiling for Rust applications
+
 ### Profiling a Rust Rideshare App with Pyroscope
+
+![Image](https://github.com/user-attachments/assets/9a6e50a1-b8df-4923-9632-79ace3fea216)
+
+Note: For documentation on Pyroscope's Rust integration visit our website
+for [rust push mode](https://grafana.com/docs/pyroscope/latest/configure-client/language-sdks/rust/)
```

Comment:
GitHub markdown lets you use notes, warnings, caution, etc. admonitions: https://github.com/orgs/community/discussions/16925

```suggestion
> [!NOTE]  
> For documentation on Pyroscope's Rust integration, refer to the [Rust push mode](https://grafana.com/docs/pyroscope/latest/configure-client/language-sdks/rust/) documentation.
```